js/apps/admin-ui/CONTRIBUTING.md

```diff
@@ -24,13 +26,13 @@ See the instructions in the [Keycloak server app](../keycloak-server/README.md).
 
 Now that the Keycloak sever is running it's time to run the development server for the Admin UI. This server is used to build the Admin UI in a manner that it can be iterated on quickly in a browser, using features such as [Hot Module Replacement (HMR)](https://vitejs.dev/guide/features.html#hot-module-replacement) and [Fast Refresh](https://www.npmjs.com/package/react-refresh).
 
-To start the development server run the following command:
+To start the development server run the following command from the root of the repository:
 
 ```bash
-pnpm dev
```

Comment:
This should not be changed, as running this command locally is a valid option.

Reply:
Why this one? What about the others?